app/web/src/components/RecipeItem.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import RecipeItem from './RecipeItem';
+
+describe('RecipeItem', () => {
+  const recipe = {
+    name: 'Test Recipe',
+    description: 'Test Description',
+    ingredients: [
+      { name: 'Ingredient 1' },
+      { name: 'Ingredient 2' },
+      { name: 'Ingredient 3' },
+    ],
+  };
+
+  test('renders recipe name', () => {
+    render(<RecipeItem {...recipe} />);
+    const recipeNameElement = screen.getByText('Test Recipe');
+    expect(recipeNameElement).toBeInTheDocument();
+  });
+
+  test('renders recipe description', () => {
+    render(<RecipeItem {...recipe} />);
+    const recipeDescriptionElement = screen.getByText('Test Description');
+    expect(recipeDescriptionElement).toBeInTheDocument();
```

Comment:
If it's a text that a user can read on the screen, you usually want to use `toBeVisible` instead. This ensures the text is on the DOM and visible, because it could be there but not be visible

Reply:
good point!